sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -122,7 +122,7 @@ def check_max_n_features(est, X, y):
 
     for n_features in range(max_n_features):
         transformer = SelectFromModel(
-            FixedImportanceEstimator(range(1, max_n_features + 1)),
+            FixedImportanceEstimator([1, 2, 2, 3, 3, 3, 4, 4, 4, 4]),
```

Comment:
I think we decided not to test tie-breaking policy in `SelectKBest`, but it might be worth checking what it does in practice with scores like this.